src/App.tsx

```diff
@@ -37,11 +37,14 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
+	const samlConfigContainer =
```

Comment:
I would add a different url param for wide container, and not only based on flow id
to me, it makes more sense as a better separation of concern 

small thing, a better name of the variable can be `containerClassName` and not `samlConfigContainer`, since this is what it holds, and the former is just one of its possible values

Reply:
I think it still makes sense to automatically trigger this based on saml-config though don't you think? That way when Descoper's are configuring embedded links to send their tenants the setup links, they don't have to worry about including this flag in the setup url.

Reply:
nice 👍